webpack/assets/javascripts/react_app/components/HostDetails/index.js

```diff
@@ -151,7 +157,11 @@ const HostDetails = ({
         >
           {tabs &&
             tabs.map(tab => (
-              <Tab key={tab} eventKey={tab} title={tab}>
+              <Tab
+                key={tab}
+                eventKey={tab}
+                title={slotMetadata?.translations[tab] || tab}
```

Comment:
In case `translations` was not defined and we want to return `tab`, it will not be translated, since `tab` is an internal identifier.

Reply:
if `translations` is not defined, or `translations[tab]` doesn't exist, the `tab` id  shouldn't be translated (i.e `Ansible` tab)